services/pokemon_service.go

```diff
@@ -25,3 +42,18 @@ func FindPokemonById(id int) (*models.Pokemon, error) {
 	}
 	return nil, nil
 }
+
+func (pks PokemonService) LoadPokemons() (bool, error) {
+	response, error := http.Get(viper.GetString("api.pokemon.url"))
```

Comment:
Same here for the config value, the URL could be better set as an attribute on your service struct

Reply:
In order to be able to test this type of functions that call external services I would recommend watching this article. There is an http client struct that you can mock

https://www.thegreatcodeadventure.com/mocking-http-requests-in-golang/